src/interfaces/filing-interfaces/filing-interface.ts

```diff
@@ -92,7 +92,9 @@ export interface RegistrationFilingIF {
       identifier: string
       naics?: NaicsIF
     }
-    businessAddress: BusinessAddressIF
+    offices: {
+      businessOffice: BusinessAddressIF
+    }
```

Comment:
Any change needed to `IncorporationFilingIF` on line 38?

Reply:
No its not required in incorporations. Its exclusive to Firms